### PR TITLE
MBH: attempting to start cdat web intergration

### PR DESCRIPTION
**Pull request by mattben**
updating Readme
adding to gitignore
removing esgf node scanning (esgf is currently busted)
minor bug fix, putting request in a try block

### CODE REVIEW

File: apps/web_fe/models.py [hunk 1]
```diff
@@ -32,10 +32,15 @@ class ESGFNode(models.Model):
...
+        status = True
+        try:
+            r = requests.get('http://' + self.host + '/esgf-node-manager/registration.xml', timeout=1)
+            if r.status == 200:
+                status = False
+        except Exception:
+            print 'http://' + self.host + '/esgf-node-manager/registration.xml'
 
-        if r.status_code != 200:
+        if status:
```
Comment (chaosphere2112) [inline apps/web_fe/models.py]: So if the status is "Success", status is false?

File: apps/web_fe/views.py [hunk 2]
```diff
@@ -253,7 +253,7 @@ def dashboard(request):
 
             for node in nodes:
                 node.refresh()
-
+    '''
```
Comment (chaosphere2112) [inline apps/web_fe/views.py]: Eh, just delete it for now. It's always in git.

File: apps/web_fe/models.py (hunk 1 above)
Comment (mattben) [inline apps/web_fe/models.py]: correct.

Comment (chaosphere2112) [inline apps/web_fe/models.py]: Mind switching the polarity on that for the sake of clarity? Default status to False, set to True if status is 200, check `if not status` on the if statement?

File: apps/web_fe/views.py (hunk 2 above)
Comment (mattben) [inline apps/web_fe/views.py]: Done

File: apps/web_fe/models.py (hunk 1 above)
Comment (mattben) [inline apps/web_fe/models.py]: Done

**Comment by chaosphere2112** [comment]
:+1: I'll merge as soon as travis gives its +1

**Pull request merged**